ui/src/app/edge/live/energymonitor/chart/section/consumption.component.html

```diff
@@ -0,0 +1,27 @@
+<svg:path [attr.d]="outlinePath" fill="#f4f4f4" stroke="LightGray"></svg:path>
+<svg:path *ngIf="isEnabled" [attr.d]="valuePath" [attr.fill]="color" stroke="LightGray"></svg:path>
+<svg:g *ngIf="isEnabled && energyFlow">
+    <svg:defs>
+        <svg:linearGradient [attr.id]="sectionId" [attr.x1]="energyFlow.gradient.x1" [attr.y1]="energyFlow.gradient.y1"
+            [attr.x2]="energyFlow.gradient.x2" [attr.y2]="energyFlow.gradient.y2">
+            <svg:stop offset="0%" stop-color="white" />
+            <svg:stop offset="100%" [attr.stop-color]="color" />
+        </svg:linearGradient>
+    </svg:defs>
+    <svg:polygon [attr.points]="energyFlow.points" attr.fill="url({{ url }}#{{ sectionId }})" />
+    <ng-container *ngIf="consumption">
+        <svg:polygon [@Consumption]="stateName" [attr.points]="energyFlow.animationPoints" style="fill:white" />
+    </ng-container>
+</svg:g>
+<svg:g *ngIf="isEnabled && squarePosition && square"
+    attr.transform="translate({{squarePosition.x}},{{squarePosition.y}})">
+    <svg:text [attr.x]="square.valueText.x" [attr.y]="square.valueText.y - (square.valueText.y * 0.3)"
```

Comment:
Die Berechnung des "square.valuteText.y" sollte komplett in der ts-Datei stattfinden und nicht zum Teil dort und zum Teil hier.